src/libraries/Common/tests/Extensions/TestingUtils/Microsoft.AspNetCore.Testing/src/RepeatAttribute.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    /// <summary>
+    /// Runs a test multiple times to stress flaky tests that are believed to be fixed.
+    /// This can be used on an assembly, class, or method name. Requires using the AspNetCore test framework.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = false)]
+    public class RepeatAttribute : Attribute
```

Comment:
We don't use the RepeatAttribute in dotnet/runtime and I don't think that this brings in the necessary infra to make this work.